apps/content/content/docs/contract-first.mdx

```diff
@@ -89,20 +90,24 @@ With your contract defined, implement the server logic:
 import { os, ORPCError } from '@orpc/server'
 import { contract } from 'examples/contract'
 
-export const pub /** public access */ = os.contract(contract) // Ensure every implement must be match contract
-export const authed /** require authed */ = os
-  .use((input, context, meta) => /** put auth logic here */ meta.next({}))
+export const publicProcedure = os.contract(contract) // Ensure every implement must be match contract
+export const authProcedure = os
+  .use((input, context, meta) => {
+    /** put auth logic here */
+    return meta.next({})
+  })
   .contract(contract)
 
-export const router = pub.router({
-  getting: pub.getting.handler((input, context, meta) => {
+export const router = publicProcedure.router({
+  getUser: publicProcedure.getUser.handler((input, context, meta) => {
     return {
-      message: `Hello, ${input.name}!`,
+      username: 'example',
+      avatar: 'example',
     }
   }),
 
-  post: {
-    find: pub.post.find
+  posts: {
+    getPost: publicProcedure.posts.getPost
```

Comment:
For my personal style, I prefer the original approach over duplicating `post` here. But it's up to you! The oRPC is also designed with that philosophy—removing anything unnecessary 😁.

Reply:
I think we should stick with the pluralising of nests as it stays close to REST design conventions then